fluss-common/src/main/java/com/alibaba/fluss/record/MemoryLogRecordsIndexedBuilder.java

```diff
@@ -158,6 +160,14 @@ public void resetWriterState(long writerId, int batchSequence) {
         this.batchSequence = batchSequence;
     }
 
+    public void overrideLastLogOffset(long lastLogOffset) {
+        Preconditions.checkArgument(
```

Comment:
nit: Usually, we static import `Preconditions` to improve readability. that is directly use `checkArgument(...)` without prefix class name.

Reply:
Done! This method will be removed.